test/test-applications/integrations/Samples.AspNetCoreRazorPages/Startup.cs

```diff
@@ -0,0 +1,46 @@
+using Datadog.Trace.IntegrationTests.DiagnosticListeners;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Samples.AspNetCoreRazorPages
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+#if NETCOREAPP2_1
```

Comment:
Is this app still useful to run and test on netcoreapp2.1?

Reply:
Yes, `services.AddMvc()` includes Razor Pages support in .NET Core 2.1. (`AddMvc()` is also available in later versions, but the standard approach is the dedicated `AddRazorPages()` call, which avoids loading controller-based dependencies)